test/integration/removeUnwantedIndexes.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai');
+const moment = require('moment');
+
+const esClient = require('../../lib/es/esClient');
+const removeUnwantedIndexes = require('../../lib/es/removeUnwantedIndexes');
+const sampleData = require('../resources/sample-data');
+
+const expect = chai.expect;
+const maxWaitTime = 1 * 60 * 1000;
+const orphanAlias = 'profiles-orphan';
+const index1 = `${orphanAlias}_20180117100000`;
+const index2 = `${orphanAlias}_20180117100010`;
+const index3 = `${orphanAlias}_20180117100020`;
+const index4 = `${orphanAlias}_20180117100030`;
+const index5 = `${orphanAlias}_20180117100040`;
+
+const watcherPrefix = '.watcher-history-6-';
+const monitorPrefix = '.monitoring-es-6-';
+
+function getIndexNames(prefix) {
+  const now = moment();
+  return [
+    prefix + now.format('YYYY.MM.DD'),
+    prefix + now.subtract(6, 'days').format('YYYY.MM.DD'),
+    prefix + now.subtract(2, 'days').format('YYYY.MM.DD'),
+  ];
+}
+
+const [todayWatcher, oldestWatcher, expiredWatcher] = getIndexNames(watcherPrefix);
+const [todayMonitor, oldestMonitor, expiredMonitor] = getIndexNames(monitorPrefix);
+
+async function deleteTestIndexes() {
+  await esClient.delete(index1);
+  await esClient.delete(index2);
+  await esClient.delete(index3);
+  await esClient.delete(index4);
+  await esClient.delete(index5);
+  await esClient.delete(expiredWatcher);
+  await esClient.delete(oldestWatcher);
+  await esClient.delete(todayWatcher);
+  await esClient.delete(todayMonitor);
+  await esClient.delete(oldestMonitor);
+  await esClient.delete(expiredMonitor);
+}
+
+describe('Elasticsearch Client', function test() {
+  this.timeout(maxWaitTime);
+
+  before(async () => {
+    await esClient.waitForHealthyEs();
+  });
+
+  beforeEach(async () => {
+    await deleteTestIndexes();
+  });
+
+  it('removeUnwantedIndexes should remove previous indexes with no alias', async () => {
+    // simulate creating orphaned indexes - these have the alias as a prefix and a datestamp
+    await esClient.createIndex(index1, sampleData);
+    await esClient.createIndex(index2, sampleData);
+    await esClient.createIndex(index3, sampleData);
+    await esClient.createAlias(index1, orphanAlias);
+    const result = await removeUnwantedIndexes(orphanAlias);
+    // order returned not deterministic, sort before check
+    result.sort();
+    // eslint-disable-next-line no-unused-expressions
+    expect(result.length).to.equal(2);
+    expect(await esClient.exists(index2)).to.be.false;
+    expect(await esClient.exists(index3)).to.be.false;
+    expect(await esClient.exists(index1)).to.be.true;
+  });
+
+  it('removeUnwantedIndexes should remove duplicate aliases', async () => {
```

Comment:
Can this test title be updated to make it clear the first added alias will be removed please?